layouts/partials/openapi/render-object-table.html

```diff
@@ -104,6 +104,22 @@
         */}}
         {{ $additionalProperties := partial "json-schema/resolve-allof" .additionalProperties }}
         {{ $type = delimit (slice "{string: " (partial "property-type" $additionalProperties) "}" ) "" }}
+    {{ else if reflect.IsMap .patternProperties }}
```

Comment:
could you update the comment on this `partial` to mention that it looks at `patternProperties` as well as `additionalProperties` ?